infra/los/templates/backend_ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "los.fullname" . }}-backend
+  namespace: {{ .Values.los_namespace }}
+  labels:
+    {{- include "los.labels" . | nindent 4 }}
+spec:
+  rules:
+    - host: {{ .Values.los_backend.fqdn }}
+      http:
+        paths:
+          - path: "/"
+            pathType: ImplementationSpecific
```

Comment:
Any particular reason you went with `ImplementationSpecific` over `Prefix` like the rest of our services?

Reply:
Resolved